kubespawner/objects.py

```diff
@@ -110,13 +113,6 @@ def make_pod_spec(
 
     pod.spec = V1PodSpec()
 
-    security_context = V1PodSecurityContext()
```

Comment:
Apparently they can - I'd recommend we leave the default on, and have people set an explicit securityContext on the init container if it's using an userID that isn't available in the pod (or vice versa)